web_src/js/features/repo-legacy.js

```diff
@@ -398,17 +398,14 @@ async function onEditContent(event) {
     }
   };
 
-  const saveAndRefresh = (dz, $dropzone) => {
+  const saveAndRefresh = (dz) => {
     showElem($renderContent);
     hideElem($editContentZone);
-    const $attachments = $dropzone.find('.files').find('[name=files]').map(function () {
-      return $(this).val();
-    }).get();
     $.post($editContentZone.attr('data-update-url'), {
       _csrf: csrfToken,
       content: comboMarkdownEditor.value(),
       context: $editContentZone.attr('data-context'),
-      files: $attachments,
+      files: dz.files.map((file) => file.uuid),
```

Comment:
This replacement is wrong and causes a regression, I spent many hours on debugging it ........

Reply:
Hmm, I'm pretty sure it was a array of either urls or uuids before and after the change.

Reply:
Thought maybe it is actually a array of `File` elements. Not sure what the value of a `input[type=file]` really is currently.

Reply:
No because the dropzone was abused, the `dz.files` seldom gets correctly updated.

Reply:
Ok, I would definitely prefer for dropzone to be removed at some point. Won't be too hard to render a static list of files with a "x" icon besides each.